packages/opentelemetry-exporter-jaeger/src/jaeger.ts

```diff
@@ -85,14 +88,40 @@ export class JaegerExporter implements SpanExporter {
   }
 
   /** Shutdown exporter. */
-  shutdown(): void {
-    // Make an optimistic flush.
-    this._flush();
-    // Sleeping x seconds before closing the sender's connection to ensure
-    // all spans are flushed.
-    setTimeout(() => {
-      this._sender.close();
-    }, this._onShutdownFlushTimeout);
+  shutdown(): Promise<void> {
```

Comment:
async/await

Reply:
here the same it will return the `_shuttingDownPromise` which will resolve correctly and you can call shutdown many times - all of them will be resolved correctly